packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -682,42 +682,6 @@ const clickIfClickable = async (
 	}
 };
 
-// Only for Android
-const waitIfAndroid = async () => {
-	if ( isAndroid() ) {
-		await editorPage.driver.sleep( 1000 );
-	}
-};
-
-/**
- * Content type definitions.
- * Note: Android only supports plaintext.
- *
- * @typedef {"plaintext" | "image" | "url"} ClipboardContentType
- */
-
-/**
- * Helper to set content in the clipboard.
- *
- * @param {Object}               driver      Driver
- * @param {string}               content     Content to set in the clipboard
- * @param {ClipboardContentType} contentType Type of the content
- */
-const setClipboard = async ( driver, content, contentType = 'plaintext' ) => {
-	const base64String = Buffer.from( content ).toString( 'base64' );
-	await driver.setClipboard( base64String, contentType );
-};
-
-/**
- * Helper to clear the clipboard
- *
- * @param {Object}               driver      Driver
- * @param {ClipboardContentType} contentType Type of the content
- */
-const clearClipboard = async ( driver, contentType = 'plaintext' ) => {
```

Comment:
I think we'd need to remove its usage from [this test](https://github.com/WordPress/gutenberg/blob/1f907579e2cd6b580fd4bbf32a31e9f2f8877de9/packages/react-native-editor/__device-tests__/gutenberg-editor-drag-and-drop.test.js#L10).